constants/config.ts

```diff
@@ -2,7 +2,8 @@ import { i18n } from '../utils/lang';
 
 export const DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME = 'hubspot.config.yml';
 
-export const HUBSPOT_CONFIGURATION_FOLDER = '.hubspot';
+// Changed for easier debugging.
+export const HUBSPOT_CONFIGURATION_FOLDER = 'hubspot';
```

Comment:
Is this just a temporary change?

Reply:
Temporary change, no worries.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/0559c584504dc458721cef1dced436eaf0e88862